core/src/main/scala/caliban/introspection/adt/__DeprecatedArgs.scala

```diff
@@ -1,3 +1,13 @@
 package caliban.introspection.adt
 
+import scala.runtime.AbstractFunction1
+
 case class __DeprecatedArgs(includeDeprecated: Option[Boolean] = None)
+
+// NOTE: This object extends AbstractFunction1 to maintain binary compatibility for Scala 2.13.
+// TODO: Remove inheritance in the next major version
+object __DeprecatedArgs extends AbstractFunction1[Option[Boolean], __DeprecatedArgs] {
+  val include: __DeprecatedArgs = __DeprecatedArgs(Some(true))
+
+  def apply(v: Option[Boolean] = None): __DeprecatedArgs = new __DeprecatedArgs(v)
+}
```

Comment:
Fairly annoying we have to extend `AbstractFunction1` to make MiMA happy with Scala 2.13. We can remove this for 2.6.0